resources/views/events/view.blade.php

```diff
@@ -1,290 +1,363 @@
-                @else
-                    @if (! Auth::user()->isInGroup($event->theGroup->idgroups))
-                        <a class="btn btn-tertiary" href="/group/join/{{ $event->theGroup->idgroups }}">@lang('events.follow_group')</a>
-                    @endif
-                    @if( is_object($is_attending) && $is_attending->status == 1 && $event->isUpcoming() )
-                        <button data-toggle="modal" data-target="#event-invite-to" class="btn btn-primary">@lang('events.invite_volunteers')</button>
-                    @else
-                        <a class="btn btn-primary" href="/party/join/{{ $formdata->id }}">RSVP</a>
-                    @endif
-                @endif
-            @endif
+              @endif
             @endif
           </div>
         </div>
       </div>
 
       <div class="row">
-          <div class="col-lg-12">
-            <p></p>
-            <p></p>
-          </div>
+        <div class="col-lg-12">
+          <p></p>
+          <p></p>
+        </div>
       </div>
 
-      <div class="row">
-        <div class="col-lg-4">
+      <!-- So far only upcoming events have been moved over to Vue. -->
+      @if($event->isUpcoming())
+        <div>
+          <div class="vue-placeholder vue-placeholder-large">
+            <div class="vue-placeholder-content">@lang('partials.loading')...</div>
+          </div>
 
-          <aside id="event-details" class="sidebar-lg-offset">
+          <?php
+          // We need to expand the user objects to pass to the client.  In due course this will be replaced
+          // by an API call to get the event details.
+          $expanded_attended = [];
+          foreach ($attended as $att) {
+            $thisone = $att;
+            $thisone['volunteer'] = $att->volunteer;
+            $thisone['userSkills'] = $att->volunteer->userSkills;
+            $thisone['fullName'] = $att->getFullName();
+            $thisone['profilePath'] = $att->volunteer->getProfile($att->id)->path;
+            $expanded_attended[] = $thisone;
+          }
+          $expanded_invited = [];
+          foreach ($invited as $att) {
+            $thisone = $att;
+            $thisone['volunteer'] = $att->volunteer;
+            $thisone['userSkills'] = $att->volunteer->userSkills;
+            $thisone['fullName'] = $att->getFullName();
+            $thisone['profilePath'] = $att->volunteer->getProfile($att->id)->path;
+            $expanded_invited[] = $thisone;
+          }
+
+          $expanded_hosts = [];
+          foreach ($hosts as $host) {
+            $thisone = $host;
+            $thisone['volunteer'] = $host->volunteer;
+            $expanded_hosts[] = $thisone;
+          }
+
+          $attendance_edit = (FixometerHelper::hasRole(Auth::user(), 'Host') && FixometerHelper::userHasEditPartyPermission($formdata->id, Auth::user()->id)) || FixometerHelper::hasRole(Auth::user(), 'Administrator');
```

Comment:
Name could be clearer, e.g. can_edit_attendance

Reply:
Agreed, will change.